cmake/SDL3_sound.pc.in

```diff
@@ -3,10 +3,10 @@ exec_prefix=${prefix}
 libdir=@CMAKE_INSTALL_FULL_LIBDIR@
 includedir=@CMAKE_INSTALL_FULL_INCLUDEDIR@
 
-Name: SDL2_sound
```

Comment:
Rename this file to `sdl3-sound.pc.in`, as we did for all SDL satellite libraries.

SDL_sound is not an official SDL project, but I suppose icculus won't mind :)

Reply:
Leaving this for later for now